### DISCUSSION TITLE
How to load css/scss based on different pages.

### DISCUSSION THREAD

**Discussion by hirenramoliyaiwb** [nuxt/nuxt#16824] [Questions]
### Describe the feature

I want to load some css/scss files only on some pages not globally so, is there any way to import/load conditionally 

### Additional information

- [ ] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by sevillaarvin** [comment]
Hi, does `useHead` meet your requirements?

https://v3.nuxtjs.org/api/composables/use-head#add-external-css

**Comment by hirenramoliyaiwb** [comment]
@sevillaarvin This is right but in that tailwind classes not working with the @apply keyword, and for that, I have to add that particular file into nuxt config file and that applying CSS globally in all the pages.

**Comment by TheAlexLichter** [comment]
I think this should happen automatically when using (S)CSS in the component/page only and when the content is big enough for an own file (worth doing an extra HTTP request)

**Reply by hirenramoliyaiwb** [reply]
@manniL  @danielroe  Please give an appropriate answer. Thanks for the comment.  

BTW as of now I am using like this: 
`<style src="~/assets/css/helper.scss" scoped></style>` in about page only 

But this tailwind code not working
`.btn-white {
  @apply h-10 px-6 font-semibold rounded-md border border-slate-200 text-slate-900;
}`
